Updated to allow multiple vaults to be configured in frisp.ini

micmania1 opened the pull request:
# Example frisp.ini

```
[default]
bucket=mybucket
vault=myvault
region=ap-southeast-2
profile=myprofile

[vault2]
bucket=mybucket
region=ap-southeast-2
profile=myprofile
```

Note that `vault2` doesn't have vault configured. In this case, it will use the name (vault2).

## Usage

This will use default from ini
`./frisp.php list `

This will use vault2
`./frisp.php list vault2`

hafriedlander commented:
I think this won't work if you try and use actions with further args? Like `./frisp.php read password*` will think the vault name is "password*". Or did you check & this works OK?

(Also, README.md will need updating :) ).

micmania1 commented:
I've updated it so that the vault is no longer optional (as my original PR introduced) which should address the issue raised. Also updated the readme to include the new params/structure.

hafriedlander commented on frisp.php:
```diff
@@ -106,10 +90,46 @@ function parse($final = true) {
+if (file_exists($_SERVER['HOME'].'/.aws/frisp.ini')) {
+	$options = parse_ini_file($_SERVER['HOME'].'/.aws/frisp.ini', true);
+
+	// If the vault param isn't passed, we'll defer to 'default' in our frisp.ini,
```
Comment no longer correct
